specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/examples/GetNamedProviderAtTenant.json

```diff
@@ -0,0 +1,253 @@
+{
+  "parameters": {
+    "$expand": "resourceTypes/aliases",
+    "resourceProviderNamespace": "Microsoft.Storage",
+    "api-version": "2020-06-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "namespace": "Microsoft.Storage",
+        "resourceTypes": [
```

Comment:
This is a partial API response. The full response with the full set of resource types and aliases is huge. LMK if that's a problem